src/components/ModalExport.tsx

```diff
@@ -210,25 +217,39 @@ const ModalExport = () => {
     setPublishing(false)
   }
 
+  const [advancedSettings, setAdvancedSettings] = useState(false)
+
+  /** Toggles advanced setting when Advanced CTA is clicked. */
+  const onAdvancedClick = () => setAdvancedSettings(!advancedSettings)
+
+  /** Updates checkbox value when clicked and set the appropriate value in the selected option. */
+  const onChangeLosslessCheckbox = () => setShouldIncludeMetaAttributes(!shouldIncludeMetaAttributes)
+
+  /** Created an array of objects so that we can just add object here to get multiple checkbox options created. */
+  const advancedSettingsArray: AdvancedSetting[] = [
+    {
+      id: 'lossless-checkbox',
+      onChangeFunc: onChangeLosslessCheckbox,
+      defaultChecked: true,
+      title: 'Lossless',
+      description: 'When checked, include all metaprogramming attributes such as archived thoughts, pins, table view, etc. Check this option for a backup-quality export that can be re-imported with no data loss. Uncheck this option for social sharing or exporting to platforms that do not support em metaprogramming attributes. Which is, uh, all of them.'
+    }
+  ]
+
   return (
     <Modal id='export' title='Export' className='popup'>
 
+      {/* Export message */}
       <div className='modal-export-wrapper'>
         <span className='modal-content-to-export'>{exportMessage}</span>
         <span className='modal-drop-down-holder'>
-          <img
-            src={dark ? ArrowDownWhite : ArrowDownBlack}
-            alt='Arrow'
-            height='22px'
-            width='22px'
-            style={{ cursor: 'pointer' }}
-            onClick={() => setIsOpen(!isOpen)}
-          />
+          <ChevronImg dark={dark} onClickHandle={() => setIsOpen(!isOpen)} />
           <div ref={setWrapper}>
             <DropDownMenu
               isOpen={isOpen}
               selected={selected}
               onSelect={(option: ExportOption) => {
+                setShouldIncludeMetaAttributes(!shouldIncludeMetaAttributes)
```

Comment:
I'm not sure I understand this. Selecting the export format should not affect `shouldIncludeMetaAttributes`. Can you explain what your intention was?

Reply:
Oh.. I left it by mistake. I was doing some testing. It is not needed here. Sorry about that